x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/downsample/TransportRollupIndexerAction.java

```diff
@@ -170,9 +177,12 @@ protected RollupIndexerAction.Response newResponse(
     private class Async extends AsyncBroadcastAction {
         private final RollupIndexerAction.Request request;
         private final ActionListener<RollupIndexerAction.Response> listener;
+        private final Task task;
+        private volatile boolean hasCancelled = false;
```

Comment:
Maybe rename to `hasCancelledOtherShardOperations`?

Reply:
done